public/graph.js

```diff
@@ -437,18 +440,20 @@ class GraphController {
                 .attr("class", (d, i) => receivedKeys[i])
                 .style("fill", (d, i) => color(i));
 
+            // Values to adjust x and width attributes
```

Comment:
add some details as to why we we need this

Reply:
Done!